src/components/common/Input/styles.js

```diff
@@ -3,6 +3,11 @@ import { StyleSheet } from 'react-native';
 const styles = StyleSheet.create({
   input: {
     height: 50,
+    borderColor: '#CCCCCC',
+    borderWidth: 1,
+  },
+  inputActive: {
+    borderColor: '#666666',
```

Comment:
Can we store these as constants?

Reply:
I thought about it and I decided against it. This file already has a hardcoded color and I think that the reasoning behind it is that these styles are just placeholders and are very likely to change so just having them here makes it easier for the dev to remove and implement the real styling. But I am open to suggestions.

Reply:
Got it, seems a good call